go.mod

```diff
@@ -9,13 +9,12 @@ require (
 	github.com/gin-contrib/cors v1.3.1
 	github.com/gin-gonic/gin v1.7.4
 	github.com/golang-jwt/jwt/v4 v4.0.0
-	github.com/golang/protobuf v1.5.2
```

Comment:
@rebeccabilbro and now it's gone … I'll make sure I've rebased correctly, but I'm pretty sure I have.